File: fizzbuzz.py
#!/usr/bin/python3

for i in range(1,35):
    # Build output string
    output=""

    if i % 5 == 0:
        output+="Fizz"
    if i % 3 == 0:
        output+="Buzz"

    # Format padded number
    i_padded=f"{i}"
    if len(f"{i}") == 1:
        i_padded=f" {i}"

    print(f"{i_padded}: {output}")

File: fizzbuzz.c
#include <stdio.h>
#include <string.h>

int main(void)
{
    
    for(int i=1; i<=100; i++)
    {
        char msg[8] = "";
        if((i%3)==0) 	 strcat(msg,"Fizz");
        if((i%5)==0)	 strcat(msg,"Buzz");
	printf("number= %d %s\n", i, msg);

    }

    return 0;
}
